modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,205 @@
+
++
+--
+[source,json]
+----
+"{
+    ""Version"": ""2012-10-17"",
+    ""Statement"": [
+        {
+            ""Effect"": ""Allow"",
+            ""Principal"": {
+                ""Federated"": ""arn:aws:iam::{AWS_AccountNo}: <1>
+
+                {Cluster_OIDC_Endpoint}"" <2>
+            },
+            ""Action"": ""sts:AssumeRoleWithWebIdentity"",
+            ""Condition"": {
+                ""StringEquals"": {
+                    ""{Cluster's OIDC Endpoint}:sub"": [
+                        ""system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-manager"",
+                        ""system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-controller-cluster""
+                    ]
+                }
+            }
+        }
+    ]
+}"
+----
+<1> Replace '{AWS_AccountNo}' with your AWS account number.
+<2> Replace '{Cluster_OIDC_Endpoint}' with the OIDC DNS identified in
+step 1b.
+
+[NOTE]
+====
+Do not include the 'https' portion of the OIDC DNS url when replacing '{Cluster_OIDC_Endpoint}' with the OIDC DNS identified in
+step 1b.  Only the alphanumeric information that proceeds the '.net/' within the URL is needed.
+====
+--
+. Add the tags necessary for subnet discovery:
+
+.. Add the following {Key: Value} tag to the VPC hosting the {product-title} (ROSA) cluster, replacing '{Cluster Infra ID}' with the Infra ID specified in step 1a:
+
+
+* kubernetes.io/cluster/${Cluster Infra ID}:owned
+
+.. Add the following Elastic Load Balancing (ELB) {Key: Value} tags to the private subnets and (optional) public subnets:
+
+* Private subnets: kubernetes.io/role/internal-elb:1
+* Public subnets: kubernetes.io/role/elb:1
++
+[NOTE]
+====
+Internet-facing and internal load balancers will be created within the  Availability Zones (AZ) of which these subnets belong.
+====
++
+For more information on adding tags to AWS resources (including VPCs and subnets), see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tag your Amazon EC2 resources].
++
+. Log into the {product-title} (ROSA) cluster as the Cluster-Admin and create a new project using the following command:
++
+[source, terminal]
+----
+$ oc new-project aws-load-balancer-operator
+----
++
+. Link the AWS IAM role created in step 2a with the AWS Load Balancer (ALB) Operator service account and create a Kubernetes Secret for the AWS Load Balancer (ALB) Operator:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+stringData:
+  credentials: |
+    [default]
+    role_arn = <aws-load-balancer-operator-role-arn><1>
+    web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+----
+<1> Replace <aws-load-balancer-operator-role-arn>' with the AWS IAM role created in step 2a.
++
+. Create the Red Hat-supported AWS Load Balancer (ALB) Operator:
```

Comment:
BTW, user can install ALBO via Console>OperatorHub as well. see https://docs.openshift.com/container-platform/4.13/networking/aws_load_balancer_operator/install-aws-load-balancer-operator.html

Reply:
@lihongan,
   Hmmmm....okay, I may have been confused on that issue... I thought for STS clusters we could not use the install option in the Operator Hub? Maybe I misinterpreted. @bchandra-ocp, Can you confirm that we can (or cannot) use the OperatorHub in this case (to install the ALBO on an STS cluster)? 
![image](https://github.com/openshift/openshift-docs/assets/122639474/0f28ec2e-2e5d-4251-a1dc-dd073705386c)

Reply:
Just the prerequisite is different for non-STS and STS cluster.  Fon non-STS cluster user should follow the prerequisite in above page and create credentialrequest, then the secret will be provisioned automatically.
For STS cluster user need to prepare the secret manually, and it is already mentioned here before this step so user can continue the installation from OperoatorHub.

Reply:
> Just the prerequisite is different for non-STS and STS cluster. Fon non-STS cluster user should follow the prerequisite in above page and create credentialrequest, then the secret will be provisioned automatically. For STS cluster user need to prepare the secret manually, and it is already mentioned here before this step so user can continue the installation from OperoatorHub.

I am still confused about how the OperatorHub can be used for installing the ALBO based on this prerequisite: 
![image](https://github.com/openshift/openshift-docs/assets/122639474/4fb10e8b-38e2-48cf-a2ae-eb46ab81ac7f)
If a user has "_An existing Red Hat OpenShift Service on AWS cluster with BYO-VPC installed in STS mode_", can they or can they not use the OperatorHub to install the ALBO. If they cannot, then should the OperatorHub be mentioned in this doc? I wouldn't think so. Please see this comment (which leads me to believe the OperatorHub does not apply to this doc). https://github.com/openshift/openshift-docs/pull/62273#discussion_r1303382121.